src/engine/source/indexerconnector/src/indexerConnector.cpp

```diff
@@ -218,16 +219,20 @@ IndexerConnector::IndexerConnector(const IndexerConnectorOptions& indexerConnect
                     continue;
                 }
 
+                const auto& finalIndexName = parsedData.contains("indexName")
```

Comment:
I believe this field is not part of the ECS. It is metadata and should not be used to decide where an event will be indexed.

Reply:
> I believe this field is not part of the ECS

I thought that we could use it internally.

>  It is metadata and should not be used to decide where an event will be indexed.

It could be a feature, to allow the user define the index and not depend on the time the indexer pushes the event.
But I'll remove it.

Reply:
Now, the date is replaced in each bulk.

Reply:
Great!